contrib/examples/actions/echochain.meta.yaml

```diff
@@ -10,4 +10,11 @@ runner_type: "action-chain"
 entry_point: "chains/echochain.yaml"
 
 enabled: true
-parameters: {}
+parameters:
+  skip_notify:
```

Comment:
It's a runner parameter. So with CLI you have to use it as a string. It's unfortunate.